docs/docsite/rst/community/collection_contributors/collection_integration_about.rst

```diff
@@ -0,0 +1,156 @@
+
+Integration test example
+-------------------------
+
+Let's say we want to test the ``postgresql_user`` module invoked with the ``name`` parameter. We expect that the module will both create a user based on the provided value of the ``name`` parameter and will report that the system state has changed. We cannot rely on only what the module reports. To be sure that the user has been created, we query our database with another module to see if the user exists.
+
+.. code-block:: yaml
+
+  - name: Create PostgreSQL user and store module's output to the result variable
+    postgresql_user:
+      name: test_user
+    register: result
+
+  - name: Check the module returns what we expect
+    assert:
+      that:
+        - result is changed
+
+  - name: Check actual system state with another module, in other words, that the user exists
+    postgresql_query:
+      query: SELECT * FROM pg_authid WHERE rolename = 'test_user'
+    register: query_result
+
+  - name: We expect it returns one row, check it
+    assert:
+      that:
+        - query_result.rowcount == 1
+
+
+Details about integration tests
+--------------------------------
+
+The basic entity of an Ansible integration test is a ``target``. The target is an :ref:`Ansible role <playbooks_reuse_roles>` stored in the ``tests/integration/targets`` directory of the collection repository. The target role contains everything that is needed to test a module.
+
+The names of targets contain the module or plugin name that they test. Target names that start with ``setup_`` are usually executed as dependencies before module and plugin targets start execution. See :ref:`collection_creating_integration_tests` for details.
+
+To run integration tests, we use the ``ansible-test`` utility that is included in the ``ansible-core`` and ``ansible`` packages. See :ref:`collection_run_integration_tests` for details. After you finish your integration tests, see to :ref:`collection_quickstart` to learn how to submit a pull request.
+
+.. _collection_integration_prepare:
+
+Preparing for integration tests
+================================
+
+To prepare for developing integration tests:
+
+#. :ref:`Set up your local environment <collection_prepare_environment>`.
+
+#. Determine if integration tests already exist.
+
+  .. code-block:: bash
+
+      ansible-test integration --list-targets
+
+
+If a collection already has integration tests, they are stored in ``tests/integration/targets/*`` subdirectories of the collection repository.
+
+If you use ``bash`` and the ``argcomplete`` package is installed with ``pip`` on your system, you can also get a full target list.
+
+.. code-block:: shell
+
+  ansible-test integration <tab><tab>
```

Comment:
```suggestion
  ansible-test integration <tab></tab>
```

Reply:
My suggested change was based on my basic knowledge of HTML, lol.
I just read the docs only to see that's actually a command used to obtain a full list of bash users in Ansible. Getting to learn about this more from my mistake.